Gemfile

```diff
@@ -18,11 +18,13 @@ gem 'foundation_rails_helper'
 # to use debugger
 gem 'byebug', group: [:development, :test]
 gem 'pry', group: [:development, :test]
+gem 'rb-readline', group: [:development, :test]
```

Comment:
I don't think this is needed. Less gems means less problems most of the time :-)